### ISSUE TITLE
Equipment gained or lost does not add up

### ISSUE THREAD

**Issue by yasonk** [leedrake5/Russia-Ukraine#6]
I could be way off, but looks like the chart labeled "Equipment gained or lost" is not correct.
It is showing 600 gain for Ukraine, but my rough calculation is closer to 300.
Ukraine total lost equipement = 700.
Ukraine captured roughly 1000.
So total gain should be 1000-700 = 300.

**Comment by qwertie** [comment]
Yeah, it looks mighty suspicious, but I'm not familiar with R. Anyone who does know R want to look for mistakes [over here](https://github.com/leedrake5/Russia-Ukraine/blob/182be4bdb1f94247f528d9968ad36fec587f5ebe/R/Automation/losses.r#L956)?

**Comment by leedrake5** [comment]
I think the discrepancy comes from adjusting for the captures. 

Adjusted total = Country A total losses - Country A captures from Country B

To elaborate, all losses by Oryx (destroyed, captured, abandoned, or damaged) are real for Country A. But Country A has also captured some equipment from Country B, so this is discounted from their total, creating the adjusted total. Captures are then measured against this adjustment. I hope this helps to understand the calculation. @qwertie identified the relevant code - if you would prefer to adjust it you are welcome to give it a go.
